Client/Frontend/Browser/Browser.swift

```diff
@@ -349,6 +353,27 @@ class Browser: NSObject {
             self.screenshotUUID = NSUUID()
         }
     }
+
+    @available(iOS 9, *)
+    func toggleDesktopSite(on: Bool) {
```

Comment:
I moved the actual user agent setting and reloading to the `Browser` class since the `BrowserViewController` class is already quite large and this logic could be considered internal to the `Browser`.